recipes/python-statuspage/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set url_version = "0.3-pre" %}
+{% set version = url_version.replace("-", "") %}
+
+package:
+  name: python-statuspage
+  version: {{ version }}
+
+source:
+  fn: python-statuspage-{{ version }}.tar.gz
+  url: https://github.com/jayfk/statuspage/archive/{{ url_version }}.tar.gz
+  sha256: 51f6de360b3fffe6a1eac1f9d3ee08ab65adef972d272634f4564b1815e50c8d
+  patches:
+    # Fix setuptools dependency.
+    # --------------------------
+    # A few requirements are needed by setuptools.
+    # In particular, `packaging` and `six`. Here
+    # we add them to PyInstaller's hidden imports
+    # to ensure they are packaged in the final
+    # bundle. See PR link below for more details.
+    #
+    # https://github.com/jayfk/statuspage/pull/52
+    #
+    - PR_52.diff
+
+build:
+  number: 0
+  skip: true  # [win or py3k]
+  script:
+    - pyinstaller -p "${PREFIX}/lib:${SP_DIR}" statuspage.spec               # [unix]
+    - pyinstaller -p "%LIBRARY_BIN%:%LIBRARY_LIB%:%SP_DIR%" statuspage.spec  # [unix]
+    - cp dist/statuspage "${PREFIX}/bin/"                                    # [unix]
+    - chmod +x "${PREFIX}/bin/statuspage"                                    # [unix]
+    - cp dist\\statuspage "%LIBRARY_BIN%"                                    # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - packaging
+    - pyinstaller
+    - pygithub
+    - click
+    - jinja2
+    - tqdm
+    - six
```

Comment:
This is a weird package. Basically it wraps all of this into one executable binary. Hence no run-time requirements. However, that is how PyInstaller works.